chain/vm/invoker.go

```diff
@@ -200,3 +205,30 @@ func DumpActorState(code cid.Cid, b []byte) (interface{}, error) {
 
 	return rv.Elem().Interface(), nil
 }
+
+// TODO: ActorUpgrade: Support versioning
+// TODO: Doesn't work well for methods that have params as params (Init's Exec, multisig Propose, etc.)
+func DumpParams(code cid.Cid, method abi.MethodNum, b []byte) (interface{}, error) {
```

Comment:
I'd do that by moving https://github.com/filecoin-project/lotus/blob/master/cli/state.go#L1202-L1211 next to https://github.com/filecoin-project/lotus/blob/master/chain/stmgr/utils.go#L614